ivadomed/scripts/prepare_dataset.py

```diff
@@ -0,0 +1,144 @@
+import argparse
+import ivadomed.utils as imed_utils
+import ivadomed.preprocessing as imed_preprocessing
+import nibabel as nib
+import numpy as np
+import os
+import scipy
+import skimage
+
+
+# normalize between 0 and 1.
```

Comment:
a few of these functions could go in ivadomed package and be exposed as APIs, e.g. normalize, gaussian_kernel, etc.

maybe an `ivadomed.math`? `ivadomed.labels`, etc.

Reply:
could be done in another PR--

Reply:
This function already exists in `ivadomed.transforms.py`: `rescale_values(arr, minv, maxv)`